src/content/en/tools/lighthouse/audits/contrast-ratio.md

```diff
@@ -0,0 +1,57 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Background and Foreground Colors Have Sufficient Contrast Ratio" Lighthouse audit.
+
+{# wf_updated_on: 2017-01-20 #}
+{# wf_published_on: 2017-01-20 #}
+
+# Background and Foreground Colors Have Sufficient Contrast Ratio  {: .page-title }
+
+## Why the audit is important {: #why }
+
+Some users have low-contrast vision. They don't see many areas that are very
+bright or very dark. Everything looks about the same brightness. If the text
+doesn't have a high contrast ratio, these users literally have trouble seeing
+it at all.
+
+## How to pass the audit {: #how }
+
+To find and fix each of element that does not have a sufficient contrast ratio:
+
+1. Install [ChromeLens][CL].
```

Comment:
@alice just curious if you think it's better to use ChromeLens or would you prefer folks use the Chrome Accessibility DevTools extension. Long term, is this functionality that you think will be brought into Chrome DevTools itself?